### PR TITLE
Add support for converting heading to strong text

### PR DESCRIPTION
**Pull request by crimelabs786**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -46,7 +47,24 @@ async function run() {
           }
           markdown.content = toMarkdown({
             ...mdast,
-            children: mdast.children.slice(1),
+            children: mdast.children
+              .slice(1)
+              .reduce((acc: Content[], child) => {
+                const newChild: Content = child;
+                if (child.type === "heading") {
```
Comment (crimelabs786) [inline src/index.ts:54]: This structure is a bit complex.

Say, this is the `heading` object (`depth` field denotes header level)

```js
{
  type: 'heading',
  depth: 3,
  children: [
    {
      type: 'text',
      value: 'Is beating the market the only goal in stock picking?',
      position: {
        start: { line: 28, column: 5, offset: 3416 },
        end: { line: 28, column: 58, offset: 3469 }
      }
    },
    [length]: 1
  ],
  position: {
    start: { line: 28, column: 1, offset: 3412 },
    end: { line: 28, column: 58, offset: 3469 }
  }
}
```

Then corresponding `boldface` text would be something like this (ignore the offsets and columns)

```ts
{
  type: 'paragraph',
  children: [
    {
      type: 'strong',
      children: [
        {
          type: 'text',
          value: 'Is beating the market',
          position: {
            start: { line: 30, column: 3, offset: 3473 },
            end: { line: 30, column: 24, offset: 3494 }
          }
        },
        [length]: 1
      ],
      position: {
        start: { line: 30, column: 1, offset: 3471 },
        end: { line: 30, column: 26, offset: 3496 }
      }
    },
    [length]: 1
  ],
  position: {
    start: { line: 30, column: 1, offset: 3471 },
    end: { line: 30, column: 26, offset: 3496 }
  }
}
```

Comment (zulienthegreat) [reply src/index.ts:54]: Can you extract this into a separate function in util?
It would be easy to modify it later.

Comment (crimelabs786) [reply src/index.ts]: Done @zulienthegreat

**Review by zulienthegreat — APPROVED** [review]
LGTM

**Pull request merged**